### PR TITLE
[Sleek] added my own color scheme wich is inspired by the dracula color scheme

### PR DESCRIPTION
**Pull request by skelletondude**
(need some improvements)

### CODE REVIEW

File: Sleek/color.ini [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+button             = 8bc34a ;
+button-active      = 98da4b ;
+button-disabled    = 353535 ;
+sidebar-button     = 8bc34a ;
+play-button        = 8bc34a ;
+tab-active         = 303030 ;
+notification       = 242424 ;
+notification-error = 242424 ;
+playback-bar       = 8bc34a ;
+misc               = FFFFFF ;
```
Comment (morpheusthewhite) [inline Sleek/color.ini:22]: Why did you add semicolon at the end?

File: Sleek/color.ini [hunk 2]
```diff
@@ -251,6 +251,29 @@ notification-error = 131417
 playback-bar       = dcd88c
 misc               = FFFFFF
 
+[Dracula]
```
Comment (morpheusthewhite) [inline Sleek/color.ini:254]: Add a preview in the theme README and in THEMES.md

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

File: Sleek/color.ini (hunk 1 above)
Comment (skelletondude) [reply Sleek/color.ini:22]: srry it was for my own comment but forgot to remove it before upload

**Comment by skelletondude** [comment]
im not very good with git etc sorry

**Comment by morpheusthewhite** [comment]
You still need to add the preview to THEMES.md. Also, add it in alphabetically sorted order (both in THEMES.md and README.md)

**Comment by morpheusthewhite** [comment]
Also, the big play button under the playlist name is not visible, you should fix it

**Comment by skelletondude** [comment]
Im going to fix all this and reopen a pull request :)
